specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/capacityReservation.json

```diff
@@ -873,6 +873,12 @@
           "type": "string",
           "description": "A unique id generated and assigned to the capacity reservation by the platform which does not change throughout the lifetime of the resource."
         },
+        "platformFaultDomainCount": {
```

Comment:
@ArcturusZhang  there is a ["platformFaultDomainCount" property set in virtualmachinescaleset.json ](https://github.com/Azure/azure-rest-api-specs/blob/feature/cplat-2022-08-01/specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/virtualMachineScaleSet.json#L4739) but it is not readonly. 
can an object with same name defined here but with readonly tag?